ui/shared/status/StatusChatInputImageArea.qml

```diff
@@ -0,0 +1,70 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import "../../imports"
+import "../../shared"
+
+Rectangle {
+    id: imageArea
+    height: 72
+
+    signal imageRemoved()
+    property url imageSource: ""
+    color: "transparent"
+    
+    Image {
+        id: chatImage
+        property bool hovered: false
+        height: 64
+        anchors.left: parent.left
+        anchors.leftMargin: 8
+        anchors.top: parent.top
+        anchors.topMargin: 9
```

Comment:
Can this be `8`? Seems like it would be more balanced with the left margin and we can use a Style prop for it